packages/dataviews/src/view-list.tsx

```diff
@@ -37,13 +64,14 @@ function ListItem( {
 	mediaField,
 	primaryField,
 	visibleFields,
-} ) {
+}: ListViewItemProps ) {
 	const itemRef = useRef( null );
 	const labelId = `${ id }-label`;
 	const descriptionId = `${ id }-description`;
 
 	useEffect( () => {
 		if ( isSelected ) {
+			// @ts-ignore
 			itemRef.current?.scrollIntoView( {
```

Comment:
If we provide the type of the value in the ref, the ignore can be removed and we'll get more helpful types:

```suggestion
	const itemRef = useRef< HTMLElement >( null );
	const labelId = `${ id }-label`;
	const descriptionId = `${ id }-description`;

	useEffect( () => {
		if ( isSelected ) {
			itemRef.current?.scrollIntoView( {
```